### DISCUSSION TITLE
[20 BETA] Very small selection area in sounds

### DISCUSSION THREAD

**Discussion by lxwndr** [linuxdeepin/developer-center#3021] [Features Request & Ideas | 特性请求 & 头脑风暴]
## Describe the bug
To select, it is necessary to click exactly inside the circle. With the mouse, this selection is not ergonomic.

Allow the user to click on any area of the bar to select. And the circle acts only as a selection indicator.

## Screenshots
![Screen Capture_select-area_20200428010334](https://user-images.githubusercontent.com/45263160/80446135-5e103000-88ec-11ea-97d8-0b052bb491b7.png)

### Distro & Version
deepin 20 BETA

### Related package version
dde-control-center 5.0.33

### Additional context
I sent this suggestion directly to Shule; but I don't know if he ignored

**Comment by lxwndr** [comment]
The problem persists in the new version: deepin 20.1 (1010) / OsBuild=11031.108

**Comment by justforlxz** [comment]
I will talk with manager for switch play and set area.

**Comment by meiyixiang** [comment]
61/5000 
This suggestion is more reflected in the interaction level. I have recorded the problem. But right now we are working hard on the desktop environment of V23 and the next release will be uniformly optimized for this kind of interaction

**Comment by meiyixiang** [comment]
This suggestion is accepted, and optimization will be scheduled later
